mmv1/products/parametermanager/product.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+---
+name: 'ParameterManager'
+display_name: 'Parameter Manager'
+versions:
+  - name: 'ga'
```

Comment:
It looks like this API is in a Pre-GA state, so we shouldn't add it to the GA provider at this point.

Reply:
Agreed, let's add this to beta provide only for now.